src/amo/components/Addon/index.js

```diff
@@ -153,58 +146,29 @@ export class AddonBase extends React.Component {
     }
   }
 
-  componentWillUnmount() {
-    const {
-      isPreviewingTheme,
-      resetThemePreview,
-      themePreviewNode,
-    } = this.props;
-
-    if (isPreviewingTheme && themePreviewNode) {
-      resetThemePreview(themePreviewNode);
-    }
-  }
-
-  onClick = (event) => {
-    this.props.toggleThemePreview(event.currentTarget);
-  }
-
-  headerImage({ compatible }) {
+  headerImage() {
     const {
       addon,
-      getBrowserThemeData,
       i18n,
-      isPreviewingTheme,
-      installStatus,
     } = this.props;
+    const title = addon ? i18n.sprintf(
+      i18n.gettext('Preview of %(title)s'),
+      { title: addon.name }
+    ) : '';
     const previewURL = addon ? addon.previewURL : null;
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
     const iconUrl = getAddonIconUrl(addon);
 
     if (type === ADDON_TYPE_THEME) {
-      const label = isPreviewingTheme ? i18n.gettext('Cancel preview') : i18n.gettext('Tap to preview');
       const imageClassName = 'Addon-theme-header-image';
-      const headerImage = <img alt={label} className={imageClassName} src={previewURL} />;
+      const headerImage = <img alt={title} className={imageClassName} src={previewURL} />;
 
       return (
         <div
           className="Addon-theme-header"
           id="Addon-theme-header"
-          data-browsertheme={getBrowserThemeData()}
```

Comment:
@bobsilverberg  - that is my understanding - that we only need it for install per @muffinresearch's comments above. 

Also I am not sure - initially I thought this might be replaced by the unification work that will be happening here #4899.

Reply:
You'll want to check, but IIRC clicking the image results in a install. So you will need the data there if that's the case.

Reply:
#4899 is just about unifying the preview images so that when static and lightweight themes are shown side by side they don't look ood. Once we're fully migrated to static themes it will be possible to plan removal of all lightweight theme code.

Reply:
> You'll want to check, but IIRC clicking the image results in a install. So you will need the data there if that's the case.

Click to install definitely isn't working today. But it did in the past so it looks like it regressed at some point. See also https://github.com/mozilla/addons-frontend/issues/613